shorts.py

```diff
@@ -0,0 +1,22 @@
+def should_I_put_shorts_on(temp):
+    if temp > 20:
```

Comment:
So 20 is too cold for shorts? You could change this to `if temp >= 20:` to include 20.